packages/compass-aggregations/src/components/focus-mode/focus-mode-stage-preview.spec.tsx

```diff
@@ -86,6 +86,18 @@ describe('FocusModeStagePreview', function () {
       const preview = screen.getByTestId('focus-mode-stage-preview');
       expect(within(preview).getByText(/no preview documents/i)).to.exist;
     });
+    it('renders missing search index text for $search', async function () {
```

Comment:
super NIT feel free to ignore, any reason not to `for searchStage of [$search, $vectorSearch, ...]` this one?

Reply:
good idea. I'll change.